pyglidein/submit.py

```diff
@@ -232,8 +232,9 @@ def write_glidein_part(self, f, local_dir=None, glidein_tarball=None, presigned_
                 if not os.path.isfile(script_path):
                     raise Exception("Stard cron script not found: {}".format(script))
                 self.write_line(f, 'cp %s %s' % (script_path, script))
-        f.write('exec env -i CPUS=$CPUS GPUS=$GPUS MEMORY=$MEMORY DISK=$DISK WALLTIME=$WALLTIME '
-                'DISABLE_STARTD_CHECKS=$DISABLE_STARTD_CHECKS ')
+        f.write('env -i CPUS=$CPUS GPUS=$GPUS MEMORY=$MEMORY DISK=$DISK WALLTIME=$WALLTIME '
+                'DISABLE_STARTD_CHECKS=$DISABLE_STARTD_CHECKS '
+                'TMPDIR=$TMPDIR TEMP=$TEMP TMP=$TMP ')
```

Comment:
Regarding TMPDIR and variations, the intention was to not use the host's default values, and only allow values configured in the glidein.  So the `env` would clear the host values out, and then `CustomEnv` could add these values back if necessary.

I guess if not having these set is an issue, we really should set them by default to the current directory where the job is running.

Reply:
Agreed, using `CustomEnv ` works well